dynacred/src/config.ts

```diff
@@ -0,0 +1,61 @@
+import * as toml from 'toml';
+
+/**
+ * Readonly container for the configuration
+ *
+ * TODO preciser name
+*/
+export class Config {
+    constructor(
+        readonly addresses: Addresses,
+    ) {}
+
+    /**
+    * Type containing both the addresses of the endpoints for servers and for
+    * clients
+    */
+    private class Addresses {
+        constructor(
+            readonly servers: string[],
+            readonly clients: string[],
+        ) {}
+    }
+}
+
+/**
+ * Load the Config from the server
+ */
+export default async function LoadConfig(): Promise<Config> {
+    const res = await fetch(window.location.origin + '/assets/conodes.toml');
+    if (!res.ok)
+        return Promise.reject(new Error(`on fetch config: ${res.status}: ${res.body}`));
+
+    const parsed = toml.parse(await res.text());
+    if (!('servers' in parsed))
+        return Promise.reject(new Error(`on parse config: unable to find "servers"`));
+    if (!(Symbol.iterator in parsed.servers))
+        return Promise.reject(new Error(`on parse config: "servers" is not iterable`));
+
+    let addrs_client:string[] = [], addrs_server:string[] = [];
```

Comment:
This is still `const` in javascript, even if you `push` into it...

Reply:
Done (handled by tslint).